docs/integration.rst

```diff
@@ -1241,6 +1243,129 @@ argument in the ``efibootmgr`` call above:
   type=ext4
   bootname=system1
 
+Raspberry Pi firmware
+~~~~~~~~~~~~~~~~~~~~~
+
+The Raspberry Pi firmware supports fail-safe OS updates with A/B booting thanks
+to the optional configuration file `autoboot.txt
+<https://www.raspberrypi.com/documentation/computers/config_txt.html#autoboot-txt>`_
+and the one-shot reboot flag `tryboot
+<https://www.raspberrypi.com/documentation/computers/raspberry-pi.html#fail-safe-os-updates-tryboot>`_.
+
+.. warning:: The Raspberry Pi firmware relies on a set of FAT filesystems to
+   sync its state. Consequently, the ``barebox`` and ``uboot`` backends, that
+   write their state in raw (i.e. outside from filesystems), are rather more
+   reliable candidates than the ``raspberrypi`` backend.
```

Comment:
Please use [Semantic Line Breaks](https://sembr.org/) in the docs.

Reply:
It was in my TODO list. I did not know that semantic, I hope it have fixed it correctly.

Please tell me if I missed some  line breaks.

Reply:
Thanks, looks good now. SemBr helps avoid noise in git diffs.